### PR TITLE
COS-254 Upgrade liquibase core version. Fix lock service exception

### PR DESCRIPTION
**Pull request by DaimonCool**

### CODE REVIEW

File: liquibase-couchbase/pom.xml [hunk 1]
```diff
@@ -257,42 +257,46 @@
...
-                            <mainClass>fully.qualified.MainClass</mainClass>
-                        </manifest>
-                    </archive>
-                    <descriptorRefs>
-                        <descriptorRef>jar-with-dependencies</descriptorRef>
-                    </descriptorRefs>
-                </configuration>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.4.1</version>
```
Comment (konstantin-umanets) [inline liquibase-couchbase/pom.xml:287]: Could be extracted to property

**Review by konstantin-umanets — APPROVED** [review]

**Review by tigrankhojoyan — APPROVED** [review]

File: liquibase-couchbase/src/main/java/liquibase/ext/couchbase/lockservice/LockStatus.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package liquibase.ext.couchbase.lockservice;
+
+public enum LockStatus {
+    LOCKED_BY_SERVICE,
+    LOCKED_BY_ANOTHER_SERVICE,
+    NO_LOCK
+
```
Comment (romchellis) [inline liquibase-couchbase/src/main/java/liquibase/ext/couchbase/lockservice/LockStatus.java:7]: Actually that much better, but i would offer small name improvement according to java convention lock names
```suggestion
public enum LockStatus {
    FREE,
    OWNER,
    NOT_OWNER
    }
```

Comment (DaimonCool) [reply liquibase-couchbase/src/main/java/liquibase/ext/couchbase/lockservice/LockStatus.java:7]: Agree. Will change it

Comment (DaimonCool) [reply liquibase-couchbase/src/main/java/liquibase/ext/couchbase/lockservice/LockStatus.java:4]: Done

**Review by romchellis — APPROVED** [review]

**Review by konstantin-umanets — APPROVED** [review]

**Pull request merged**